ChaosMod/Effects/db/Player/PlayerGTA2.cpp

```diff
@@ -0,0 +1,40 @@
+#include <stdafx.h>
+
+/*
+* Effect by kolyaventuri
+*/
+
+static const float height = 35.f;
+static const float speedFactor = 1.f + (0.5f / (180.f / 2.236936f));
+static float baseFov;
+static Cam camera;
+
+static void OnStart() {
+	Ped player = PLAYER_PED_ID();
+	Vector3 rot = GET_ENTITY_ROTATION(player, 2);
+
+	camera = CREATE_CAM("DEFAULT_SCRIPTED_FLY_CAMERA", 1);
+	ATTACH_CAM_TO_ENTITY(camera, player, 0.f, 0.f, height, true);
```

Comment:
This most likely will change the camera position, once the player is not standing up straight, right? like riding on a bike for example (not tested)
maybe fixed, if the last flag is switched to false

Reply:
oh quite possibly. I'll play around with that. That might explain why the camera goes all wonky when you get run over, or die